### ISSUE TITLE
Bundled products subscription options not correlated with main product subscription options

### ISSUE THREAD

**Issue by benjaminmoquet** [Prospress/woocommerce-subscribe-all-the-things#424]
When a Product Bundle (using WooCommerce Product Bundles) has a subscription, and the Simple Products inside the bundle also have subscription options, the subcriptions prices are not correlated.

For instance I have a product bundle PROD1 with 2 differents subscription options : 
10€ for 3 months and 8€ for 6 months.

I have a simple product called PROD2 in my product bundle PROD1 with same subscriptions options lengths (but differents prices) : 2€ for 3 months and 1€ for 6 months.

Boths products have "overide regular price" and "force subscription" in options.

On my main product page PROD1, PROD2 is set to 0€ whatever subscription plan i choose. 

Should subscription prices of both products be correlated ?

**Comment by benjaminmoquet** [comment]
one more precision, PROD2 is price indivually.

**Comment by franticpsyx** [comment]
@benjaminmoquet in a Bundle with the subscription plans setup you described, PROD2 should be listed with the price you defined under **Product Data > General**. The subscription plans of a Bundle always override the subscription plans of its "children". 

Note that when you choose **Price > Override product** for a Bundle the price you define there only overrides the base price of the Bundle. Child item prices are not overridden and if a child item has a price override in a subscription plan it will get ignored, even if there's only one plan like in your case.

In short, when you add PROD2 inside a Bundle, PROD2's price in the Bundle will not get pulled from an equivalent subscription plan (assuming one exists) created in PROD2.

**Comment by benjaminmoquet** [comment]
ok thank you very much. I guess we'll have to make a custom development for this :(
